feature-libs/product-configurator/common/components/configurator-cart-entry-bundle-info/configurator-cart-entry-bundle-info.component.html

```diff
@@ -0,0 +1,53 @@
+<ng-container *ngIf="orderEntry$ | async as orderEntry">
+  <ng-container *ngIf="isBundleBasedConfigurator(orderEntry)">
+    <ng-container *ngIf="(numberOfLineItems$ | async) > 0">
```

Comment:
Can we remove this container and add it to the container above to reduce the number of nesting HTML components
`*ngIf="isBundleBasedConfigurator(orderEntry) && (numberOfLineItems$ | async) > 0"`

Reply:
Yes, it is possible to merge both containers but your point regarding reducing complexity of the nesting HTML is wrong because `ng-container` never creates an additional element in the HTML tree

Reply:
Yes, it would not reduce the number of DOM elements but it would reduce the number of elements in the HTML template which makes it more readable.

Reply:
That's why you followed your advice and merge the conditions